src/khoj/database/management/commands/serve_generated_images_from_server.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+
+from khoj.database.models import Conversation
+from khoj.utils.helpers import ImageIntentType, is_none_or_empty
+
+
+class Command(BaseCommand):
+    help = "Serve Khoj generated images from a different URL."
+
+    def add_arguments(self, parser):
+        # Pass Source URL
+        parser.add_argument(
+            "--source",
+            action="store",
+            help="URL from which generated images are currently served.",
+        )
+        # Pass Destination URL
+        parser.add_argument("--destination", action="store", help="URL to serve generated image from going forward.")
+
+        # Add a new argument 'reverse' to the command
+        parser.add_argument(
+            "--reverse",
+            action="store_true",
+            help="Revert to serve generated images from source instead of destination URL.",
+        )
+
+    def handle(self, *args, **options):
+        updated_count = 0
+        if not options.get("source") or not options.get("destination"):
+            self.stdout.write(
+                self.style.ERROR(
+                    "AWS_IMAGE_UPLOAD_BUCKET environment variable or --source, --destination args needs to be set."
```

Comment:
the `AWS_IMAGE_UPLOAD_BUCKET` isn't being used in the script. Would this be more accurate?
```suggestion
                    "Set both --source, --destination args to respective URLs"
```